# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mlview/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur mlview-0.9.0-orig/src/mlview-editor.cc mlview-0.9.0/src/mlview-editor.cc
--- mlview-0.9.0-orig/src/mlview-editor.cc	2005-08-07 07:42:28.000000000 +0000
+++ mlview-0.9.0/src/mlview-editor.cc	2006-04-05 18:42:26.742256000 +0000
@@ -157,7 +157,7 @@
 			contextual_menu (NULL)
 	{}
 
-	static void EditorPriv::schemas_window_destroy_cb
+	static void schemas_window_destroy_cb
 	(GtkWidget *a_widget, struct DocumentWindowData *a_win) ;
 
 	static void validation_window_destroy_cb
diff -Nur mlview-0.9.0-orig/src/mlview-plugin.h mlview-0.9.0/src/mlview-plugin.h
--- mlview-0.9.0-orig/src/mlview-plugin.h	2005-08-07 07:47:54.000000000 +0000
+++ mlview-0.9.0/src/mlview-plugin.h	2006-04-05 18:41:46.339731000 +0000
@@ -46,12 +46,12 @@
 		
 		//forbid assignation/copy
 		Plugin& operator= (Plugin const&) ;
-		Plugin::Plugin (Plugin const&) ;
+		Plugin (Plugin const&) ;
 
 	public:
 		Plugin (const PluginDescriptor &a_descr) ;
 
-		Plugin::Plugin (const UString &a_url) ;
+		Plugin (const UString &a_url) ;
 		
 		virtual ~Plugin () ;
 		
diff -Nur mlview-0.9.0-orig/src/mlview-source-view.h mlview-0.9.0/src/mlview-source-view.h
--- mlview-0.9.0-orig/src/mlview-source-view.h	2005-08-07 07:42:28.000000000 +0000
+++ mlview-0.9.0/src/mlview-source-view.h	2006-04-05 18:37:22.583247250 +0000
@@ -53,7 +53,7 @@
 
 	enum MlViewStatus set_default_options () ;
 
-	MlViewXMLDocument* SourceView::get_document () ;
+	MlViewXMLDocument* get_document () ;
 
 	GtkUIManager * get_ui_manager () ;
 
diff -Nur mlview-0.9.0-orig/src/mlview-tree-view.h mlview-0.9.0/src/mlview-tree-view.h
--- mlview-0.9.0-orig/src/mlview-tree-view.h	2005-07-22 19:14:28.000000000 +0000
+++ mlview-0.9.0/src/mlview-tree-view.h	2006-04-05 18:37:03.422049750 +0000
@@ -178,7 +178,7 @@
 
 	enum MlViewStatus build_app_edit_menu () ;
 
-	GtkDialog * TreeView::get_expand_tree_dialog ()  ;
+	GtkDialog * get_expand_tree_dialog ()  ;
 
 	void clear_completion_popup_submenus () ;
 
